File: backend/src/main/java/de/rubeen/bsc/entities/web/EventEntity.java
package de.rubeen.bsc.entities.web;


import org.joda.time.DateTime;

public class EventEntity {
    private String subject;
    private DateTime startTime, endTime;

    public EventEntity(String subject, DateTime startTime, DateTime endTime) {
        this.subject = subject;
        this.startTime = startTime;
        this.endTime = endTime;
    }


    public String getSubject() {
        return subject;
    }

    public DateTime getStartTime() {
        return startTime;
    }

    public DateTime getEndTime() {
        return endTime;
    }
}

File: backend/src/main/kotlin/de/rubeen/bsc/entities/web/EventEntity.kt
package de.rubeen.bsc.entities.web


import org.joda.time.DateTime

class EventEntity {
    var subject: String? = null
    var startTime: DateTime? = null
    var endTime: DateTime? = null

    constructor(subject: String?, startTime: DateTime, endTime: DateTime) {
        this.subject = subject
        this.startTime = startTime
        this.endTime = endTime
    }

    constructor()
}

